junifer/api/res/run_conda.zsh

```diff
@@ -0,0 +1,23 @@
+#!/bin/zsh
```

Comment:
would it be viable to have the `res` as a plain `txt` containing a text with variable placeholders that get replaced by fill values for a specific shell or are the scripts for the different shell vastly different? just wondering if its desirable avoiding 1 script per shell.

Reply:
We would need the file to be created on-demand for that and not have it as a file. I wanted a single file but for simplicity made it separate.